TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/MecanumDriveBase.java

```diff
@@ -29,43 +29,59 @@ public MecanumDriveBase(HardwareMap hardwareMap, boolean autonomous){
         lf.setDirection(DcMotor.Direction.FORWARD);
         rf.setDirection(DcMotor.Direction.REVERSE);
         lb.setDirection(DcMotor.Direction.FORWARD);
+        rb.setDirection(DcMotor.Direction.REVERSE);
 
         // reset encoders
         lf.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
         rf.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
         lb.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
+        rb.setMode(DcMotor.RunMode.RUN_WITHOUT_ENCODER);
 
         // Run Without Encoders
         if (!autonomous)
         {
-        lf.setMode(runmode);
-        rf.setMode(runmode);
-        lb.setMode(runmode);
-        rb.setMode(runmode);
+            lf.setMode(runmode);
+            rf.setMode(runmode);
+            lb.setMode(runmode);
+            rb.setMode(runmode);
         }
         else
         {
             lf.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
             rf.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
             lb.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
-            rb.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
+            rb.setMode(DcMotor.RunMode.RUN_WITHOUT_ENCODER);
```

Comment:
This is probably wrong.  Everything is setMode(STOP_AND_RESET_ENCODER) except rb, which gets (RUN_WITHOUT_ENCODER).